pkgs/development/python-modules/azure-iot-device/default.nix

```diff
@@ -0,0 +1,57 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  pythonOlder,
+
+  setuptools,
+
+  urllib3,
+  deprecation,
+  paho-mqtt,
+  requests,
+  requests-unixsocket2,
+  janus,
+  pysocks,
+  typing-extensions,
+}:
+buildPythonPackage rec {
+  pname = "azure-iot-device";
+  version = "2.14.0";
+  pyproject = true;
+
+  disabled = pythonOlder "3.8";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "azure_iot_device";
+    hash = "sha256-ttSNSTLCQAJXNqzlRMTnG8SaFXasmY6h3neK+CSW/84=";
+  };
+
+  nativeBuildInputs = [ setuptools ];
+
+  propagatedBuildInputs = [
+    urllib3
+    deprecation
+    paho-mqtt
+    requests
+    requests-unixsocket2
+    janus
+    pysocks
+    typing-extensions
+  ];
+
+  doCheck = false;
```

Comment:
Please leave a comment why we cannot run checks.

Reply:
Fixed